redi-connect-front/src/components/organisms/Footer.tsx

```diff
@@ -1,83 +1,95 @@
 import React from 'react'
-import { Columns, Container, Section } from 'react-bulma-components'
+import { Element, Columns, Container, Section, Heading } from 'react-bulma-components'
 import { useTranslation } from 'react-i18next'
+import { ReactComponent as RediSchool } from '../../assets/images/redi-school-logo.svg'
+import { ReactComponent as Deloitte } from '../../assets/images/deloitte.svg'
 import Icons from '../atoms/MediaIcons'
 import './Footer.scss'
 
 const RediFooter = () => {
   const year = new Date().getFullYear()
   const { t } = useTranslation()
 
+  const supportLinks = [
+    {
+      url: 'https://connect.redi-school.org/downloadeables/redi-connect-code-of-conduct.pdf',
+      name: t('footer.faq')
+    },
+    {
+      url: 'mailto:career@redi-school.org',
+      name: t('footer.contact')
+    },
+    {
+      url: 'https://www.redi-school.org',
+      name: t('footer.name')
+    }
+  ]
+
+  const legalLinks = [
+    {
+      url: 'https://www.redi-school.org/berlin-transparency/',
+      name: t('footer.transparency')
+    },
+    {
+      url: 'https://www.redi-school.org/imprint/',
+      name: t('footer.cookies')
+    },
+    {
+      url: 'https://www.redi-school.org/data-privacy-policy',
+      name: t('footer.privacy')
+    }
+  ]
+
   return (
-    <footer className="footer">
+    <footer className="footer footer--border">
       <Section>
         <Container>
-          <Columns>
-            <Columns.Column className="is-hidden-mobile">
-              <p>
-                <a href="https://www.redi-school.org/">{t('footer.name')}</a>
-              </p>
-              <p>&copy; {year} {t('footer.copyright')}</p>
+          <Columns breakpoint="mobile">
+            <Columns.Column mobile={{ size: 'full' }}>
+              <Element>
```

Comment:
Please don't add empty divs to solve styling problems. The right solution is to declare an element a block here. (hope I understood the requirements right)

Reply:
Not empty anymore since an url was added to the element